working/m12-strings-english.txt

```diff
@@ -1281,11 +1532,11 @@
 ^L1284^[FC FF 07 00][82 FF _L3559_][00 FF]
 ^L1285^[D4 FF 19 00]@[9F FF] discharged[01 FF]  a very stinky gas![00 FF]
 ^L1286^[86 FF _L1314_][68 FF]@You cannot grasp the true form[01 FF]  of [9F FF]' attack![F0 FF 31 00 00 00][1B FF 78 00][F0 FF 36 00 00 00][D4 FF 51 00][1B FF 28 00][D4 FF 51 00][00 FF]
-^L1287^[FC FF 2A 00][C7 FF][69 FF][1B FF 28 00]@I understand you guys are already[01 FF]  claiming to be heroes.[22 FF][01 FF]@Well, it is a gazillion years too early for[01 FF]  you to oppose Giygas![22 FF][01 FF]@You must feel pretty stupid to keep[01 FF]  fighting without even knowing[22 FF][01 FF]@what Giygas looks like![22 FF][01 FF]@If you were to ever see Giygas, you'd[01 FF]  be so petrified with fear,[22 FF][01 FF]@you'd never be able to run away![22 FF][01 FF]@...That's how scary it is![22 FF][01 FF]@So, do you want me to turn off the[01 FF]  <Devil's Machine?>[22 FF][01 FF]@Well, prepare to be amazed![D4 FF 4A 00][1B FF 96 00][C7 FF][FC FF 2B 00][69 FF][00 FF]
+^L1287^[FC FF 2A 00][C7 FF][69 FF][1B FF 28 00]@I understand you guys are [01 FF]  already claiming to be heroes.[22 FF][01 FF]@Well, it is a gazillion years too [01 FF]  early for you to oppose Giygas![22 FF][01 FF]@You must feel pretty stupid to [01 FF]  keep fighting without even[01 FF]  knowing what Giygas looks like![22 FF][01 FF]@If you were to ever see[01 FF]  Giygas, you'd be so petrified[22 FF][01 FF]  with fear, you'd never be able[01 FF]  to run away![22 FF][01 FF]@...That's how scary it is![22 FF][01 FF]@So, do you want me to turn off [01 FF]  the <Devil's Machine?>[22 FF][01 FF]@Well, prepare to be amazed![D4 FF 4A 00][1B FF 96 00][C7 FF][FC FF 2B 00][69 FF][00 FF]
 ^L1288^@[C2 FF][1A FF 05 00] tried to get away,[1B FF 3C 00][02 FF]@and did![02 FF][00 FF]
 ^L1289^@[9F FF] relaxed[01 FF]  and took a deep breath.[02 FF][00 FF]
 ^L1290^@The Suporma sang the song[01 FF]  <Ode to Orange Kid.>[02 FF]@As soon as it finished, the machine[01 FF]  broke down.[6D FF FF 00 94 00][1D FF][00 FF]
-^L1291^@By using the Insignificant Present, you[01 FF]  had a very fruitful experience[02 FF]@that cannot be understood by someone[01 FF]  who does not use something[01 FF]  insignificant.[1D FF][00 FF]
+^L1291^@By using the Insignificant [01 FF]  Present, you had a very fruitful[01 FF]  experience, that cannot be[01 FF]  understood by someone who[01 FF]  does not use something[01 FF]  insignificant.[1D FF][00 FF]

```

Comment:
This line is missing an 02 FF after it got fixed.

Reply:
ok.